vsphere/datadog_checks/vsphere/config.py

```diff
@@ -94,6 +95,12 @@ def __init__(self, instance, init_config, log):
         self.tags_prefix = instance.get("tags_prefix", DEFAULT_VSPHERE_TAG_PREFIX)
         self.should_collect_attributes = is_affirmative(instance.get("collect_attributes", False))
         self.collect_property_metrics = is_affirmative(instance.get("collect_property_metrics", False))
+        if sys.version_info[0] >= 3:
```

Comment:
I _think_ we can remove all these checks now https://github.com/DataDog/integrations-core/pull/18417

Reply:
Okay, I'll try removing the checks for Python version, and see if the CI passes.